---a/net/minecraft/network/login/INetHandlerLoginClient.java
+++b/net/minecraft/network/login/INetHandlerLoginClient.java
@@ -6,8 +6,8 @@
 import net.minecraft.network.login.server.S02PacketLoginSuccess;
 import net.minecraft.network.login.server.S03PacketEnableCompression;
 
-public interface INetHandlerLoginClient extends INetHandler
-{
+public interface INetHandlerLoginClient extends INetHandler {
+
     void handleEncryptionRequest(S01PacketEncryptionRequest packetIn);
 
     void handleLoginSuccess(S02PacketLoginSuccess packetIn);
@@ -15,4 +15,4 @@
     void handleDisconnect(S00PacketDisconnect packetIn);
 
     void handleEnableCompression(S03PacketEnableCompression packetIn);
-}
+}
